sdk/test/resource/resource_test.cc

```diff
@@ -173,13 +174,21 @@ TEST(ResourceTest, OtelResourceDetector)
     }
   }
   EXPECT_EQ(received_attributes.size(), expected_attributes.size());
+#if defined(_WIN32)
+  putenv("OTEL_RESOURCE_ATTRIBUTES=");
```

Comment:
I think if you compile at "Warnings Level 4" in IDE, you'd see the warning here. There's another place in the actual code (not in tests) that also has the same issue.

Reply:
I logged an issue for the other one - #655

Reply:
The same as above, `_putenv` doesn't trigger CRT security warning.

Reply:
Ok, I'm now wondering what is the purpose of `putenv_s` at all 😄